www/python/src/templates/preview.html

```diff
@@ -0,0 +1,269 @@
+				width: 83%;
+			}  
+		}	
+
+		@media (max-width:575px) {
+			.review-article-listing li .reviews-article-info {
+				width: 85%;
+			}
+
+			.review-article-row {
+				padding-bottom: 15px;
+			}
+
+			.review-article-row h2 {
+				font-size: 20px;
+			}
+
+			.preview-top-heading span {
+				font-size: 13px;
+			}
+		}
+
+		.aricle-preview-top-block .preview-top-heading {
+    position: fixed;
+    width: 100%;
+    top: 0;
+}
+
+.review-article-row {
+    padding-top: 24px;
+}
+
+@media (max-width:1199px) {
+	.article-preview-media-row iframe {
+    	max-width: 100%;
+	}
+}
+    </style>	
+  </head>
+  <body class="bg-white">
+    
+
+    <div class="main-wrapper article-preview-page">
+        <div class="aricle-preview-top-block">
+        	<div class="preview-top-heading">
+        		<div class="container-fluid">
+        			<span>Das ist eine Vorschau, die nicht bearbeitet werden kann</span>
+        		</div> 	
+        	</div>
+
+        	<div class="review-article-row">
+        		<div class="container">
+	        		<!-- <h2>This article needs {{ problems|length }} reviews</h2> -->
+	        		<h2>Dieser Artikel benötigt {{ problems|length }} Überarbeitungen</h2>
+	        		<input type="hidden" id="query-url" value="{{ query }}">
+	        		<!-- <ul class="review-article-listing"> 
+	        			<li><a href="#"><img src="assets/img/review-media1.png"> <div class="reviews-article-info"><h5>Überarbeiten</h5><p>In diesem Artikel wurde ein Link automatisch aktualisiert. Prüfe, ob der Link noch stimmt.</p></div></a></li>
+
+	        			<li><a href="#"><img src="assets/img/review-media2.png"> <div class="reviews-article-info"><h5>Lückenhaft</h5><p>In diesem Artikel muss grundlegend etwas verändert werden. Schau nach, was fehlt und überarbeite den Artikel.</p></div></a></li>
+
+	        			<li><a href="#"><img src="assets/img/review-media3.png"> <div class="reviews-article-info"><h5>Veraltet</h5><p>In diesem Artikel fehlen Belege. Recherchiere fehlende Quellen und füge sie ein.</p></div></a></li>
+	        		</ul> -->
+	        		<ul class="review-article-listing">
+	        			{% for item in problems %}
+	        				{% if(item in image_prob) %}
+		        				<li>
+		        					<a href="">
+		        						<img src="{{ url_for('static', filename=image_prob[item]['image']) }}">
+		        						<div class="reviews-article-info">
+		        							<h5>{{ item }}</h5>
+		        							<p>
+		        								{{ image_prob[item]['text'] }}
+		        							</p>
+		        						</div>
+		        					</a>
+		        				</li>
+		        			{% endif %}
+	        			{% endfor %}
+	        		</ul>
+        		</div>
+        	</div>	
+        </div>	
+
+        <div class="article-preview-media-row">
+        	<div class="container">
+        		<!-- <img src="https://render-tron.appspot.com/screenshot/{{ query }}/?width=1536&height=754" class="img-fluid"> -->
+        		<iframe src="{{ query }}" style="height:621px;width:1260px;"></iframe>

```

Comment:
I noticed the page shown in this iframe can be interacted with. I can actually edit articles in here. The text on top of the iframe says otherwise. This appears to be a bug. I assume the iframe should be read-only.